zipline/testing/fixtures.py

```diff
@@ -792,12 +804,33 @@ class WithBcolzEquityDailyBarReaderFromCSVs(WithBcolzEquityDailyBarReader):
     _write_method_name = 'write_csvs'
 
 
-class WithEquityMinuteBarData(WithTradingEnvironment):
+def _trading_days_for_minute_bars(calendar,
+                                  start_date,
+                                  end_date,
+                                  lookback_days):
+    first_session = calendar.minute_to_session_label(start_date)
```

Comment:
This was moved from https://github.com/quantopian/zipline/pull/1381/files/3784a2cf55b9c89b4912bd48ceaf5237e75edb6a#diff-9691651cd6b02428961a23d1a1049526L857

For parity, it may make sense to make the same call on the `end_date`.